L/Lua/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+using BinaryBuilder
+
+name = "Lua"
+version = v"5.3.5"
+
+sources = [
+    "https://www.lua.org/ftp/lua-5.3.5.tar.gz" =>
+        "0c2eed3f960446e1a3e4b9a1ca2f3ff893b6ce41942cf54d5dd59ab4b3b058ac",
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/lua-*
+
+if [[ ${target} == *-apple-* ]]; then
+    MAKE_TARGET=macosx
+elif [[ ${target} == *-freebsd* ]]; then
+    # NOTE: The FreeBSD ports build also uses the bsd instead of the freebsd target,
+    # since it hard-codes fewer things, apparently
+    MAKE_TARGET=bsd
+elif [[ ${target} == *-mingw* ]]; then
+    MAKE_TARGET=mingw
+else
+    MAKE_TARGET=linux
+fi
+
+# XXX: Work around Lua apparently not caring that Windows uses .exe
+if [[ ${target} == *-mingw* ]]; then
+    TO_BIN="lua.exe luac.exe"
+else
+    TO_BIN="lua luac"
+fi
+
+make -j${nproc} ${MAKE_TARGET}
+make install INSTALL_TOP=${prefix} INSTALL_LIB=${libdir} TO_BIN=${TO_BIN}
```

Comment:
Always protect your variables
```suggestion
make install INSTALL_TOP="${prefix}" INSTALL_LIB="${libdir}" TO_BIN="${TO_BIN}"
```